extensions/amp-story-shopping/amp-story-shopping.md

```diff
@@ -145,6 +145,10 @@ Sets the color of the CTA button and drawer.
 <img alt="theme" src="https://user-images.githubusercontent.com/3860311/161297644-f25837e0-82ec-4960-a219-edfaf7d09fd9.png"
 layout="intrinsic" width="800" height="135">
 
+#### `cta-text` {string} optional
+
+Sets the Call To Action Text for the shopping attachment page outlink.
```

Comment:
We need to mention that it's a button and what the default is.
This can be symmetric with the copy in the [page-attachment docs](https://github.com/processprocess/amphtml/blob/main/extensions/amp-story-page-attachment/amp-story-page-attachment.md#cta-text-optional).

`String that customizes the call to action button text. The default is "Shop now".`